ethcore/src/tx_filter.rs

```diff
@@ -100,16 +117,135 @@ mod test {
 	use spec::Spec;
 	use client::{BlockChainClient, Client, ClientConfig, BlockId};
 	use miner::Miner;
-	use ethereum_types::Address;
+	use ethereum_types::{U256, Address};
 	use io::IoChannel;
 	use ethkey::{Secret, KeyPair};
 	use super::TransactionFilter;
 	use transaction::{Transaction, Action};
 	use tempdir::TempDir;
 
-	/// Contract code: https://gist.github.com/arkpar/38a87cb50165b7e683585eec71acb05a
+	/// Contract code: https://gist.github.com/VladLupashevskyi/84f18eabb1e4afadf572cf92af3e7e7f
 	#[test]
 	fn transaction_filter() {
+		let spec_data = r#"
```

Comment:
We are trying to move such string data into external resources and use in the code via: let spec_data = include_str!("../res/filename.json");